src/sumologic-context-manager/timers.spec.ts

```diff
@@ -0,0 +1,163 @@
+import * as api from '@opentelemetry/api';
+import { SumoLogicContextManager } from './index';
+
+window.setImmediate = (callback: any) => {
+  setTimeout(callback);
+};
+
+const NativeSetTimeout = window.setTimeout;
+const NativeSetInterval = window.setInterval;
+const NativeSetImmediate = window.setImmediate;
+const NativeRequestAnimationFrame = window.requestAnimationFrame;
+const NativeQueueMicrotask = window.queueMicrotask;
```

Comment:
Wow, I didn't realize this API is so widely available and that it has been for some time. Time flies. 🙂